### ISSUE TITLE
Not consuming all of output triggers fatal error

### ISSUE THREAD

**Issue by tjconcept** [mafintosh/csv-parser#61]
``` sh
$ echo "a
1
2" | csv-parser | head -n 1
{"a":"1"}
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: write EPIPE
    at exports._errnoException (util.js:1026:11)
    at WriteWrap.afterWrite (net.js:799:14)
```

Not consuming the entire input seems to trigger an error.

**Comment by baptistemanson** [comment]
This issue should be closed, right? :)

**Comment by tjconcept** [comment]
Why? I think it is still an issue?

**Comment by shellscape** [comment]
This is going to be an issue with `EPIPE`. #57 addressed a portion of that, but it was in with some other changes that we couldn't accept. If the author doesn't circle back to that PR, we'll reimplement the fix for that in `v2.0`.

**Comment by shellscape** [comment]
Closing as stale/abandoned.
